.circleci/config.yml

```diff
@@ -23,20 +23,21 @@ jobs:
       - run:
           name: Update Carthage
           command: brew outdated carthage || (brew uninstall carthage --force; HOMEBREW_NO_AUTO_UPDATE=1 brew install carthage --force-bottle)
```

Comment:
#nth would be to drop all this homebrew stuff for build time but that's more invasive. Happy to leave as is and hope for Carthage to find a way forward.

Reply:
Yeah I'd like to keep the door open for re-enabling Carthage at some point in the future